### ISSUE TITLE
PEP 3105 Make print a function in docs/core/benchmarks

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#8406]
|[<img alt="rodrigc's avatar" src="https://avatars.githubusercontent.com/u/1895943?s=50" width="50" height="50">](https://github.com/rodrigc)| @rodrigc reported|
|-|-|
|Trac ID|trac#8406|
|Type|enhancement|
|Created|2016-06-03 21:16:07Z|
|Branch|https://github.com/twisted/twisted/tree/8406-rodrigc-print|

This ticket was split out of [ticket:5812] which was too big to review.

PEP 3105 ( ​​​https://www.python.org/dev/peps/pep-3105/ ) **print** deprecates has changed from a statement to a function.

In Python 2.6 and Python 2.7, the default is print as a statement

```
print "Hello"
```

In Python 3.x, the above example is an error, since only print as a function is available.

To consistently use **print** as a function
on Python 2.6, Python 2.7, and Python 3.x, this should be done:

```
from __future__ import print_function

print("Hello")
```

<details><summary>Searchable metadata</summary>

```
trac-id__8406 8406
type__enhancement enhancement
reporter__rodrigc rodrigc
priority__normal normal
milestone__Python_3_x Python-3.x
branch__8406_rodrigc_print 8406-rodrigc-print
branch_author__ 
status__closed closed
resolution__fixed fixed
component__conch conch
keywords__None None
time__1464988567412267 1464988567412267
changetime__1466007385643407 1466007385643407
version__None None
owner__rodrigc rodrigc
cc__rodrigc@...
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="rodrigc's avatar" src="https://avatars.githubusercontent.com/u/1895943?s=50" width="50" height="50">](https://github.com/rodrigc)<a name="note_1"></a>|@rodrigc commented|
|-|-|

See:
https://github.com/twisted/twisted/pull/122

**Comment by twisted-trac** [comment]
|[<img alt="adiroiban's avatar" src="https://avatars.githubusercontent.com/u/204609?s=50" width="50" height="50">](https://github.com/adiroiban)<a name="note_2"></a>|@adiroiban set owner to @rodrigc|
|-|-|

Looks good. Will merge. Thanks!

**Comment by twisted-trac** [comment]
|[<img alt="adiroiban's avatar" src="https://avatars.githubusercontent.com/u/204609?s=50" width="50" height="50">](https://github.com/adiroiban)<a name="note_3"></a>|@adiroiban commented|
|-|-|

wrong branch

**Comment by twisted-trac** [comment]
|[<img alt="adiroiban's avatar" src="https://avatars.githubusercontent.com/u/204609?s=50" width="50" height="50">](https://github.com/adiroiban)<a name="note_4"></a>|@adiroiban set status to `closed`|
|-|-|

In changeset 1e3d50441a0de30d10028f810cc357b1b740c19f
```
#!CommitTicketReference repository="" revision="1e3d50441a0de30d10028f810cc357b1b740c19f"
Merge 8406-rodrigc-print: Use print function in docs/core/benchmarks.

Author: rodrigc
Reviewer: adiroiban
Fixes: #8406
```
